src/components/Bar.svelte

```diff
@@ -51,6 +51,13 @@
     this.set({ config: Object.assign({}, this.get('config'), {apps}) })
   }
 
+  async function updateSettingsURIs () {
+    const baseURI = await stack.get.settingsBaseURI()
+
+    // Assume all Settings' links are all located in first index
```

Comment:
Since it's expressed in the mockups, it's probably not mandatory. But I agree, we may find a way to be more robust to prevent any code-break… I'll think about it and make some update :+1: